spec/services/podcasts/get_episode_spec.rb

```diff
@@ -80,9 +80,10 @@
 
   it "schedules a Create job when an episode doesn't exist" do
     allow(podcast).to receive(:existing_episode).and_return(nil)
+
     expect do
       described_class.new(podcast).call(item: item)
-    end.to have_enqueued_job.on_queue("podcast_episode_create") # .with(podcast.id)
+    end.to change { PodcastEpisodes::CreateWorker.jobs.size }.by(1)
```

Comment:
What do you think about changing this up a little bit to be something like:

```ruby
sidekiq_assert_enqueued_with(job: PodcastEpisodes::CreateWorker, args: [podcast.id, item.to_h]) do
  described_class.new(podcast).call(item: item)
end
```
It's a little more specific by including the arguments and using the Sidekiq helpers and is essentially testing the same thing.